### ISSUE TITLE
A typo

### ISSUE THREAD

**Issue by napobo3** [tkrahn/extract23#4]
```
echo "======================================== EXRACT23 ======================================="
        echo "Usage: "
        echo "extract23.sh -b <sorted_hg19_bamfile.bam> -r <hg19_ref.fasta> [-t <23andMe_V3_hg19_ref.tab.gz>] [-o output.txt] [-v]"
        echo "  Parameters:"
```

Should be : 

```
echo "======================================== EXTRACT23 ======================================="
        echo "Usage: "
        echo "extract23.sh -b <sorted_hg19_bamfile.bam> -r <hg19_ref.fasta> [-t <23andMe_V3_hg19_ref.tab.gz>] [-o output.txt] [-v]"
        echo "  Parameters:"
```

**Comment by tkrahn** [comment]
Thank you. Fixed in version 1.2

**Comment by napobo3** [comment]
The fix was tested.
